README.md

```diff
@@ -1,58 +1,102 @@
 # invenio-subjects-lcsh
 
-LCSH subject terms for InvenioRDM
+*LCSH subject terms for InvenioRDM*
 
 Install this extension to get [Library of Congress Subject Headings](https://id.loc.gov/authorities/subjects.html) into your instance.
 
 ## Installation
 
 From your instance directory:
 
-    pipenv install invenio-subjects-lcsh
+```bash
+pipenv install invenio-subjects-lcsh
+```
 
 This will add it to your Pipfile.
 
-### Versions
+## Versions
 
-This repository follows [calendar versioning](https://calver.org/):
+This repository follows [calendar versioning](https://calver.org/) for year and month:
 
-`2021.06.18` is both a valid semantic version and an indicator of the year-month corresponding to the loaded LCSH terms.
+`2021.06.18` is both a valid semantic version and an indicator of the year-month corresponding to the loaded terms.
+`18` here is a patch number (not a day).
 
+So far the package is compatible with InvenioRDM 9.1+'s subjects "ABI". If there is a breaking change, a compatibility
+table will be provided to indicate which version is compatible with with InvenioRDM's "ABI".
 
 ## Usage
 
 There are 2 types of users for this package. Maintainers of the package and instance administrators.
 
 ### Instance administrators
 
-For instance administrators, after you have installed the extension as per the steps above, you are done. It should just work.
+For instance administrators, after you have installed the extension as per the steps above, you will want to reload your instance's fixtures: `pipenv run invenio rdm-records fixtures`. This will install the new terms in your instance.
 
-### TODO -- Maintainers
+Updating existing terms currently requires manual replacement.
```

Comment:
Is manual replacement straightforward? I am not sure what that looks like, so am just wondering if instructions could be added?